### PR TITLE
Update README to point to step-by-step gov guide.

### PR DESCRIPTION
**Pull request by jasonkb**
Also updates URLs in FL and MD gov.
And adds Jason C and Chris E to code owners.

### CODE REVIEW

File: states/FL/governor.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 contact_form:
   steps:
-    - visit: "https://www.flgov.com/contact-gov-scott/email-the-governor/"
+    - visit: "https://www.flgov.com/contact-gov-desantis/email-the-governor/"
```
Comment (csnardi) [inline states/FL/governor.yaml]: Might as well make this the even more generic https://www.flgov.com/email-the-governor/.

**Comment by jasonkb** [comment]
@csnardi Thank you for good points! Could you take another look?

File: README.md [hunk 2]
```diff
@@ -2,43 +2,57 @@
...
-  ... configs for other federal executives with web forms, as resistbot adds support ...
+  * president.yml  # for the White House
+  ... configs for other federal executives with web forms, as Resistbot adds support ...
 ```
 
 Notes:
 
 - Bio IDs come from Biographical Directory of the United States Congress
-  and from Open States. For other officials, we may invent a bio ID.
+  and from OpenStates.
```
[the remark this answers is not in the document]
Comment (jasonkb) [reply README.md]: Done

File: states/FL/governor.yaml (hunk 1 above)
Comment (jasonkb) [reply states/FL/governor.yaml]: Done

**Comment by jasonkb** [comment]
Yup happy to iterate on CODEOWNERS if Chris E doesn't want this.

**Pull request merged**